### ISSUE TITLE
About the data（didnt show anything）

### ISSUE THREAD

**Issue by scarrrr316** [sabrogden/Ditto#14]
It dont show any thing after I open ditto
![image](https://user-images.githubusercontent.com/43269183/100715551-54a2f500-33f2-11eb-840e-918959014e6f.png)
↑
like this ，how can i fix it ？
 and i want to know how to export all clips into file at once

**Comment by scarrrr316** [comment]
help ， i used sqltespy to check the database files and it shows error  ，database disk image malformed
![](https://s3.ax1x.com/2020/12/01/DhvIfS.png)
can i save my data ?? it is very important to me !!!

**Comment by AndyEr2** [comment]
I confirm this bug. As a temporary solution, I go into the settings and select the database file again. Then the program starts working correctly. This has to be done every time after restarting Windows. And I must also say that filling the database works correctly regardless of the above operation. The only problem is displaying content.

**Comment by scarrrr316** [comment]
I selected the db file again, but the problem was not solved. I also tried to reboot my computer and restart ditto .I think my problem is not display wrong ,is that it cant read the database ,the database file seems to be corrupted.

> I confirm this bug. As a temporary solution, I go into the settings and select the database file again. Then the program starts working correctly. This has to be done every time after restarting Ditto. And I must also say that filling the database works correctly regardless of the above operation. The only problem is displaying content.

**Comment by scarrrr316** [comment]
the last thing that i copied is the db file of ditto ,i tried to move it to d drive (hdd → ssd) ,to increase the reading speed

**Comment by galantra** [comment]
Was your database file renamed to _Ditto_BAD.db_? Because this just happened to me as I updated the portable version.

**Comment by scarrrr316** [comment]
> Was your database file renamed to _Ditto_BAD.db_? Because this just happened to me as I updated the portable version.

no,it didn't . it is still ditto.db ,maybe is the problem of my harddisk, i try to fix my harddsik by using chkdsk.exe , and maybe during the process, my ditto.db files damaged

**Comment by Gazeld** [comment]
Try also to load your database with the previous version of ditto and see if it works.
Else, try to open the .de file with a program, for example DB Browser for SQL Lite, and try to see if anything is wrong.

**Comment by Chealer** [comment]
@scarrrr316: Does 03.25.113 or ulterior behave the same way? If so, does searching reveal any entry (as reported in ticket #65)?

<details>
  <summary>License</summary>
This comment and any other from myself on this ticket is offered under the terms of CC0 1.0.
</details>
